otter/integration/tests/test_convergence.py

```diff
@@ -689,12 +747,11 @@ def test_group_config_update_triggers_convergence(self):
         )
 
 
-class ConvergenceSet1WithCLB(ConvergenceSet1):
+class ConvergenceSet1WithCLB(unittest.TestCase):
```

Comment:
Non-blocking: Not sure if `Set1` in the name has got any significance but `ConvergenceWithCLBTests` might be a better name?